pipelines/ops/druid/Jenkinsfile.ingestion

```diff
@@ -25,8 +25,22 @@ node("ops-slave") {
                 module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 currentWs = sh(returnStdout: true, script: 'pwd').trim()
+        // populating the vars for raw-overlord and rollup-overlord cluster respectively.
+                list_vars="$ingestion_task_names";
```

Comment:
Change to `list_vars=params.ingestion_task_names;`

Reply:
Thanks. updated as suggested.